vowpalwabbit/vwdll.cpp

```diff
@@ -219,17 +219,6 @@ VW_DLL_MEMBER size_t VW_CALLING_CONV VW_HashFeatureStaticA(const char * s, unsig
   return VW::hash_feature_static(str, u, hash, num_bits);
 }
 
-VW_DLL_MEMBER void  VW_CALLING_CONV VW_AddLabel(VW_EXAMPLE e, float label, float weight, float base)
```

Comment:
This may be breaking API change. We should probably go through a deprecation cycle for it.